Library/Homebrew/cmd/audit.rb

```diff
@@ -659,6 +659,25 @@ def audit_line(line, lineno)
     end
   end
 
+  def audit_prefix_has_contents
+    return unless formula.prefix.directory?
+
+    unless Dir.glob("#{formula.prefix}/**/*").any? do |f|
+         if File.directory?(f)
```

Comment:
New indentation looks better, but I'm still not convinced the ` end `s are lining up correctly. It's erroring out without the 3 ends at the bottom, but it really doesn't look right.